zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -0,0 +1,77 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_args = [ "-lparachain=debug" ]
+default_db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/relaychain-1964f8b557f10085cdc18f4105ad0bbb3df4c4c6.tgz?raw=true"
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "charlie"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "dave"
+  validator = true
+  args = ["--sync warp", "--reserved-only", "--reserved-nodes {{'alice'|zombie('multiAddress')}} {{'bob'|zombie('multiAddress')}} {{'charlie'|zombie('multiAddress')}}"]
+
+[[parachains]]
+id = 2000
+cumulus_based = true
+chain_spec_path = "zombienet/tests/0007-warp-sync-parachain-spec.json"
+add_to_genesis = false
+
+  [[parachains.collators]]
+  name = "dave"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-587c1ed24ddd7de05c237cf7c158fff53b8f5b26.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "eve"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-587c1ed24ddd7de05c237cf7c158fff53b8f5b26.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "ferdie"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-587c1ed24ddd7de05c237cf7c158fff53b8f5b26.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "one"
+  validator = false
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["-lsync=debug","--sync warp","--","--sync warp"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-to-sync-59f3cf1381f613f05e70e12465f13255e62c8dab.tgz?raw=true"
```

Comment:
Why do we need this db snapshot? Since both should para and internal relay chain node should warp sync, we should remove it and start from scratch. Same for two and three.